libs/ui/lib/file-input/FileInput.tsx

```diff
@@ -62,6 +62,10 @@ export const FileInput = forwardRef(
           onDragEnter={() => setDragOver(true)}
           onDragLeave={() => setDragOver(false)}
           onDrop={() => setDragOver(false)}
+          /* Validating this onBlur causes the required 
+              error message to appear on click. Instead we can
+              validate on form submit by overriding onBlur */
+          onBlur={() => {}}
```

Comment:
Any thoughts on this @david-crespo? We might want to validate `onChange` if we have anything like validating filenames or sizes – but it might give you an error when you clear the file, which we may or may not want. Presently we are only really validating required or not.

This fixes the issue where the error appears as soon as you click on the file input.

Reply:
As discussed in chat, I think validate on blur only if dirty (#1289) would be our ideal behavior, though I don't think RHF supports it yet. I will look into patching or PRing it so we can test that out. On the other hand, I don't think that would solve #1786 because you could still trigger by typing something invalid in the name field and then trying to change the radio input.

Reply:
Yeah – though this specifically is targeting the error appearing when you click on the file input. It seems like the `onBlur` is triggered whenever the file input opens.